src/pages/fieldManagement/settings/saveField/OptionEditor.jsx

```diff
@@ -42,7 +47,7 @@ export default function OptionEditor({
                 flexDirection: 'column',
                 marginBottom: 12,
               }}
-              key={option.id}
+              key={option?.created || option?.value}
```

Comment:
`value` is still not a reliable key.
1. As you edit a previously saved option's value, this key changes on every render, which causes the input to loose focus.
2. Because you are editing values, it is still possible for two previously saved options to have the same value.

Reply:
Changed in https://github.com/WildMeOrg/codex-frontend/pull/412/commits/b2c0f77857f9e4d41298476ae26b4473b092be20